backend/corpora/common/entities/collection.py

```diff
@@ -150,6 +151,96 @@ def to_dict(db_object):
 
         return results
 
+    @classmethod
+    def list_collections(cls, session: Session, visibility: str = None) -> typing.List[dict]:
+        """
+        Get a subset of columns, in dict form, for all Collections with the specified visibility. If visibility is None,
+        return *all* Collections.
+        @param session: the SQLAlchemy session
```

Comment:
change @ to : for all paramters and return

Reply:
is this a new standard for docstrings for python?

Reply:
It's a standard, not really new. It's sphinx docstring format.

Reply:
It's what we've been using, so let's keep things consistent for now.